app/schemas/org.hackillinois.android.database.Database/2.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
```

Comment:
I think we should modify the .gitignore to have the database schemas -- thoughts?